multidim-interop/js/v0.41/test/ping.spec.ts

```diff
@@ -0,0 +1,101 @@
+/* eslint-disable no-console */
+/* eslint-env mocha */
+
+import { } from 'aegir/chai'
+// import { createClient } from 'redis'
+
+import { createLibp2p, Libp2pOptions } from 'libp2p'
+// import { tcp } from '@libp2p/tcp'
+import { webSockets } from '@libp2p/websockets'
+import { noise } from '@chainsafe/libp2p-noise'
+import { mplex } from '@libp2p/mplex'
+import { yamux } from '@chainsafe/libp2p-yamux'
+import { multiaddr } from '@multiformats/multiaddr'
+
+describe('ping test', () => {
+  it('should ping', async () => {
+    let nodeFetchImport = "node-fetch"
+    let fetch = (typeof window === "undefined") ?
+      (await import(nodeFetchImport)).default :
+      window.fetch
+
+    const TRANSPORT = process.env.transport
+    const SECURE_CHANNEL = process.env.security
+    const MUXER = process.env.muxer
+    const IS_DIALER_STR = process.env.is_dialer
+    const IP = process.env.ip
+
+    const isDialer = IS_DIALER_STR === 'true'
```

Comment:
might force this to true for browser (I didn't do this,  but makes sense to me now)

Reply:
webrtc can listen, no?

Reply:
Can it? I was under the impression that this only will be possible once we support browser-to-browser, which we do not yet, do we?

Reply:
We don't do this yet. Hopefully soon